cs/src/core/Bond.cs

```diff
@@ -3,7 +3,52 @@
 
 namespace Bond
 {
-    using System.Collections.Generic;
+    using System;
+    using System.Collections.Generic;
+
+    public partial class GUID
+    {
+        public static implicit operator Guid(GUID bondGuid)
+        {
+            return BitConverter.IsLittleEndian ?
```

Comment:
Bond doesn't yet support big endian machines (nor does .NET AFAIK), so it's going to be sufficient to just have the little endian code and a test like
`Assert.IsTrue(BitConcerted.IsLittleEndian, "GUID conversion is only little endian for now.");`
so that the big endian port will break until this code is revisited.